lib/src/base.dart

```diff
@@ -125,13 +125,20 @@ class SideMenu extends StatefulWidget {
   bool get inverse => _inverse == -1;
 
   @override
+  // ignore: no_logic_in_create_state
```

Comment:
However, this line was the only one I could not refactor in my judgment, so I added a definition that ignores the warning for the moment.

The `createState` method should not have processing, so the timing for generating each of these States must be adjusted.